wordpress-mobile/WordPress-iOS
# Fixes an issue with focus in the re-login flow.

### Description:

When the auth token is lost (not sure why, but that's besides the point as it seems to be happening from time to time) and the re-login flow is triggered, there seems to be a focus issue with the popup that comes up.

The keyboard seems to be coming up on top of the popup.

### Screenshots:

It's not an issue with larger screens:

![file](https://user-images.githubusercontent.com/1836005/44593805-cb280580-a79a-11e8-9d02-09f25b1b181f.jpg)

But it prevents users with smaller screens or accessibility-changes from moving forward.

![attachment0](https://user-images.githubusercontent.com/1836005/44593836-e5fa7a00-a79a-11e8-8076-fe47d0e812bc.jpeg)

### How to reproduce the original issue:

For future reference, use version 10.7.0.2 since that version includes this problem:

1. Run the app, and make sure you're logged in.
2. Kill the app normally.
3. Change [this line](https://github.com/wordpress-mobile/WordPress-iOS/blob/10.7.0.2/WordPress/Classes/Utility/WPAuthTokenIssueSolver.m#L74) to always return true.
4. Run the App again.

### Testing:

Repeat the reproduction steps listed above in this branch.  Make sure the keyboard is not shown while the popup is up.

Important: if you're testing in the simulator make sure the virtual keyboard is up.
